src/ValidatorKeys.h

```diff
@@ -124,11 +131,7 @@ class ValidatorKeys
     sign (std::string const& data) const;
 
     /** Returns the public key. */
-    PublicKey const&
-    publicKey () const
-    {
-        return publicKey_;
-    }
+    PublicKey const &publicKey() const { return keys_.publicKey; }
```

Comment:
Nit: Formatting. Please restore the formatting of the original function (other than that extra space between `publicKey` and `()`.